starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,74 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
+
+  <meta charset="UTF-8">
+  <meta name ="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Apple Pie Recipe</title>
+  <link rel="stylesheet" href ="./css/main.css">
+
 </head>
+
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
+
+  <header>
+    <h1>Apple Pies</h1>
+  </header>
+
+  <div class="content"> 
+    <p>This was my grandmother's apple pie recipe. I have never seen another one
 quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
-
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
-
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
+prizes in local competitions. I hope it becomes one of your favorites as well!</p>
+  </div> 
+
+<div class="content image">
+  <img src="./images/recipe-info.png" alt="Image" style="height:70px">
+</div>
+
+<hr class="content"> 
+
+<div class="content ingredients">
+<h2>Ingredients</h2>
+<ul>
+<li>- 1 recipe pastry for a 9 inch double crust pie</li>
+<li>- 1/2 cup unsalted butter</li>
+<li>- 3 tablespoons all-purpose flour</li>
+<li>- 1/4 cup water</li>
+<li>- 3 tablespoons all-purpose flour</li>
+<li>- 1/4 cup water</li>
+<li>- 1/2 cup white sugar</li>
+<li>- 1/2 cup packed brown sugar</li>
+<li>- 8 Granny Smith apples - peeled, cored and sliced </li>
+</ul>
+</div>
+
+<div class="content image">
+    <img src="./images/cooking-info.png" alt="Image" style="height:100px">
+</div>
+
+<hr class="content"> 
+
+<div class="content ingredients">
+    <h2>Directions</h2>
+    <ul>
```

Comment:
Rather than hardcoding the numbers, you should use an ordered list. This will be helpful later when you're dealing with more items.

```
<ol>
<li>content</li>
<li>content</li>
</ol> 
```